source/deploying-with-ansible/roles/wazuh-opendistro.rst

```diff
@@ -1,24 +1,24 @@
-.. Copyright (C) 2021 Wazuh, Inc.
+.. Copyright (C) 2020 Wazuh, Inc.
 
-.. _ansible-wazuh-elasticsearch:
+.. _ansible-wazuh-opendistro:
 
-Elasticsearch
---------------
+Open Dstro
```

Comment:
```suggestion
Open Distro
```

Reply:
This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files